introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Solution {
+  static private final int MAX_NUMBER_OPERATIONS = 10;
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    int inputNumber = input.nextInt();
+    int result;
+    for (int i = 1; i <= MAX_NUMBER_OPERATIONS; i++) {
```

Comment:
Is it actually max number of operations or exact number of operations, as AFAICS it loops always this number of times. Then I'd suggest to call it NUMBER_OF_ITERATIONS